### PR TITLE
feat: clean up glitch assets and add files

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Could you take a look at this when you get a chance @RandellDawson and @moT01? I'm thinking of updating/prettifying the rest of the boilerplates like this. Is there anything I missed or something else we should add?

The last think I need to do is change out the favicon for a new on on our CDN. That's waiting for another PR to be accepted.

**Comment by moT01** [comment]
What do you think of [this one](https://github.com/freeCodeCamp/freeCodeCamp/issues/35541) @scissorsneedfoodtoo?

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo In `package.json`, wouldn't it be better to leave `npm start` as `node server.js` and then add a `npm run dev` that uses `nodemon` ?  Also, why does `package.json` not use 2-space indentation like the freeCodeCamp repo does?

Related to issue #35541, I like the idea of moving all instructions for the project to the challenge pages.  This means we could just have something generic for the `index.html` pages for the projects (i.e. project name, maybe a link back to the challenge).

**Comment by scissorsneedfoodtoo** [comment]
@moT01 and @RandellDawson, could you take another look at this when you get a chance?

There were a couple of usage examples that had to be removed from the landing page. I could create another quick PR to add them to the /learn page, or we could possible add them to https://github.com/freeCodeCamp/freeCodeCamp/pull/39311.

Also, if this is approved I could create a quick PR to update the demo project.

File: README.md [hunk 1]
```diff
@@ -1,22 +1,3 @@
-# API Project: URL Shortener Microservice for freeCodeCamp
+# URL Shortener Microservice
 
 [![Run on Repl.it](https://repl.it/badge/github/freeCodeCamp/boilerplate-project-urlshortener)](https://repl.it/github/freeCodeCamp/boilerplate-project-urlshortener)
```
Comment (RandellDawson) [inline README.md:3]: The bolierplate `README.md` files are eventually only going to be something like the following (per Task number 4 of https://github.com/freeCodeCamp/freeCodeCamp/issues/39721):

# [URL Shortener Microservice](https://www.freecodecamp.org/learn/apis-and-microservices/apis-and-microservices-projects/url-shortener-microservice)

Comment (scissorsneedfoodtoo) [reply README.md:3]: Thanks for pointing that out @RandellDawson. Will make the change now.

**Review by moT01 — APPROVED** [review]
LGTM 🎉 Is there any reason we would want to wait on merging this? I'm fine adding it now - my only concern was that the user stories on production aren't quite as specific as these were. We have that PR I need to fix up for them still - I made a note to myself of the ones here. Since production doesn't actually have any tests at the moment, not having these user stories wouldn't hurt too much I don't think. We could just wait on it until we get that other PR onto production. What do you two think?

**Comment by scissorsneedfoodtoo** [comment]
Hey @moT01, thanks for reviewing this again.

I'm fine either way -- there's no rush to get this in. Last I checked there isn't a PR to add tests for this yet. But I could definitely make a PR to add the user stories and any other information over on /learn. Then hopefully another contributor can come in and all they'll need to worry about is adding tests.

Does that sound good?

**Comment by Sky020** [comment]
@scissorsneedfoodtoo Are you happy to add the user stories to `/learn`?

**Comment by scissorsneedfoodtoo** [comment]
Hi @Sky020, sure, I'm happy to add these in. I have a branch with the changes ready, but haven't created a PR yet because https://github.com/freeCodeCamp/freeCodeCamp/pull/39311 is still being reviewed.

That PR seems to need a rebase, though, so I might go ahead and add the user stories there so we have all the upcoming changes in one place.

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo @Sky020 @moT01 Just to clarify the status here, is everyone in agreement this PR is good to go once the `/learn` changes hit production?

File: public/style.css [hunk 2]
```diff
@@ -1,60 +1,55 @@
...
-.footer {
-	margin-top: 40px;
-}
-
 .user-stories {
   position: relative;
   text-align: justify;
   max-width: 700px;
   margin: 15px auto;
 }
```
Comment (Sky020) [inline public/style.css:24]:
```suggestion
```

File: public/style.css [hunk 3]
```diff
@@ -1,60 +1,55 @@
...
+  list-style-type: none;
 }
 
 li {
   margin-bottom: 0.5em;
 }
+
 .user-stories li {
   margin-bottom: 1em;
 }
```
Comment (Sky020) [inline public/style.css:46]:
```suggestion
```

File: public/style.css [hunk 4]
```diff
@@ -1,60 +1,55 @@
...
-  background-color: #fff;
+  background-color: #eeeef0;
 }
+
 ol {
-	list-style-position: inside;
+  list-style-position: inside;
 }
+
 ul {
```
Comment (Sky020) [inline public/style.css:36]:
```suggestion
```

File: public/style.css [hunk 5]
```diff
@@ -1,60 +1,55 @@
...
-	list-style-type: none;
+  list-style-type: none;
 }
 
 li {
   margin-bottom: 0.5em;
 }
```
Comment (Sky020) [inline public/style.css:42]:
```suggestion
```

File: public/style.css [hunk 6]
```diff
@@ -1,60 +1,55 @@
...
 .user-stories {
   position: relative;
   text-align: justify;
   max-width: 700px;
   margin: 15px auto;
 }
+
 code {
   font-family: monospace;
   padding: 2px;
```
Comment (Sky020) [inline public/style.css:28]:
```suggestion
```

File: public/style.css [hunk 7]
```diff
@@ -1,60 +1,55 @@
...
-  color: black;
-  background-color: #fff;
+  background-color: #eeeef0;
 }
+
 ol {
```
Comment (Sky020) [inline public/style.css:32]:
```suggestion
```

File: views/index.html [hunk 8]
```diff
@@ -1,48 +1,32 @@
 <!DOCTYPE html>
 
 <html>
+  <head>
+    <title>URL Shortener Microservice</title>
```
Comment (Sky020) [inline views/index.html:5]:
```suggestion
    <title>URL Shortener Microservice | freeCodeCamp.org</title>
```

**Review by Sky020 — CHANGES_REQUESTED** [review]
Remove unused CSS
Update title to what we have been doing for other boilerplates

**Comment by Sky020** [comment]
@RandellDawson There is this matter: https://github.com/freeCodeCamp/freeCodeCamp/issues/39825

Should `mongoose` and `mongodb` be removed?

**Comment by RandellDawson** [comment]
> Should mongoose and mongodb be removed?

@moT01 I believe we want to do this, correct?

**Comment by scissorsneedfoodtoo** [comment]
Thanks for catching that @Sky020 and for all of your suggestions. I'll go ahead and commit those now.

Also, thanks for pointing that out @RandellDawson. I'll remove those packages in another commit. Just let me know if we decide to include them by default again.

**Comment by RandellDawson** [comment]
@Sky020 Do you approve of these changes now?  If so, we can merge this once the corresponding changes to the `/learn` that was merged recently is deployed to production.

**Review by Sky020 — APPROVED** [review]
LGTM 👍

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Now that the changes from https://github.com/freeCodeCamp/freeCodeCamp/pull/39311 have hit production, we can merge this PR.  Thanks @scissorsneedfoodtoo and everyone involved for getting these changes made.

**Pull request merged**